bigbluebutton-html5/imports/ui/components/recording/component.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { defineMessages, injectIntl } from 'react-intl';
+import Button from '/imports/ui/components/button/component';
+import Modal from '/imports/ui/components/modal/simple/component';
+import { styles } from './styles';
+
+const intlMessages = defineMessages({
+  startTitle: {
+    id: 'app.recording.startTitle',
+    description: 'start recording title',
+  },
+  stopTitle: {
+    id: 'app.recording.stopTitle',
+    description: 'stop recording title',
+  },
+  startDescription: {
+    id: 'app.recording.startDescription',
+    description: 'start recording description',
+  },
+  stopDescription: {
+    id: 'app.recording.stopDescription',
+    description: 'stop recording description',
+  },
+  yesLabel: {
+    id: 'app.audioModal.yes',
+    description: 'label for yes button',
+  },
+  noLabel: {
+    id: 'app.audioModal.no',
+    description: 'label for no button',
+  },
+});
+
+class RecordingComponent extends Component {
+  constructor(props) {
+    super(props);
+    const {
+      closeModal,
+      toggleRecording,
+    } = props;
+
+    this.closeModal = closeModal;
+    this.toggleRecording = toggleRecording;
+  }
+
+  render() {
+    const { intl, meeting } = this.props;
```

Comment:
We could pass only `meeting.recordProp.recording` from a service or a container rather than rerendering on any change in `meeting`